### PR TITLE
[2.7] bpo-33600: document that platform.linux_distribution() has been removed

### PR DESCRIPTION
**Pull request by timofurrer**
This change addresses the issue here: https://bugs.python.org/issue33600

Is there a better marker than `.. versionchanged::` for this? I tried to find similar situations, but couldn't find any. Maybe someone could point me to one?

https://bugs.python.org/issue33600

### CODE REVIEW

File: Doc/library/platform.rst
Comment (vstinner) [inline Doc/library/platform.rst]: I don't think that it's correct to use ".. versionchanged:: 3.8" tag in Python 2.7. Just add a regular paragraph before ".. versionadded:: 2.6".

File: Doc/library/platform.rst
Comment (vstinner) [inline Doc/library/platform.rst]: please add "Python": since Python 3.5 ... removed in Python 3.8

**Review by vstinner — CHANGES_REQUESTED** [review]

File: Doc/library/platform.rst
Comment (timofurrer) [reply Doc/library/platform.rst]: Done

File: Doc/library/platform.rst
Comment (timofurrer) [reply Doc/library/platform.rst]: Done

**Comment by timofurrer** [comment]
I have made the requested changes; please review again

**Review by vstinner — APPROVED** [review]
LGTM, but I would prefer to have a second review from a second core developer, but I'm not 100% sure that we should document this Python 3 change in the Python 2.7 doc.

**Comment by encukou** [comment]
This has some precedent in e.g. [urllib docs](https://docs.python.org/2/library/urllib.html).
Perhaps use a Note like in urllib? But this way it looks fine too, to me at least.

**Comment by vstinner** [comment]
>  Perhaps use a Note like in urllib?

I looked at the HTML output. Honestly, if you don't read carefully the whole documentation, you can easily miss the new note.

I modified the doc to use a ".. note::" and I agree that it's much easier to catch this note. So yes, please, @timofurrer: modify your PR to write the new paragraph as a note.

```
   Returns a tuple ``(distname,version,id)`` which defaults to the args given as
   parameters.  ``id`` is the item in parentheses after the version number.  It
   is usually the version codename.

   .. note::
      This function is deprecated since Python 3.5 and removed in Python 3.8.
      See alternative like the `distro <https://pypi.org/project/distro>`_
      package.

   .. versionadded:: 2.6
```

**Comment by timofurrer** [comment]
@vstinner dnoe!

image

**Comment by vstinner** [comment]
@timofurrer: Thank you Timo Furrer for your contribution! It's better as the note ;-)

Thanks @encukou for the review as well.

**Pull request merged**
